packages/block-library/src/post-template/edit.js

```diff
@@ -27,9 +27,35 @@ const TEMPLATE = [
 	[ 'core/post-excerpt' ],
 ];
 
-function PostTemplateInnerBlocks() {
+function useClassNameFromBlockContext( postId, postType ) {
+	const post = useSelect(
```

Comment:
Since we have added the field in REST API, I don't think we need this hook. We already have the `post.class_list` when we fetched the posts, so we can just add this field to `blockContexts` and pass where needed only this info.

Reply:
@ntsekouras I removed hook and add class list field to blockContexts. Should 'classList' be named as `blockContextClassList={ blockContext.classList }`?

Reply:
Thanks! It's fine as is.